### PR TITLE
Task/ees 795 cannot change footnotes when amend

### PR DESCRIPTION
**Pull request by shakes90**

### CODE REVIEW

File: src/explore-education-statistics-admin/src/pages/release/edit-release/data/ReleaseDataPage.tsx [hunk 1]
```diff
@@ -31,26 +32,35 @@ const ReleaseDataPage = () => {
...
+        });
+      },
+    );
   }, [releaseId]);
 
   useEffect(() => {
-    if (activeTab === 'footnotes') {
-      getFootnoteData();
-    }
+    getFootnoteData();
```
Comment (ntsim) [inline src/explore-education-statistics-admin/src/pages/release/edit-release/data/ReleaseDataPage.tsx:54]: Given that `getFootnoteData` is called again in Tab's `onToggle`, this could lead to a unnecessary double request.

If the tabs is already open on the Footnotes tab, then it makes sense to have it like it was before. I would probably revert this change.

File: src/explore-education-statistics-admin/src/pages/release/edit-release/data/ReleaseFootnotesSection.tsx [hunk 2]
```diff
@@ -82,22 +82,34 @@ const ReleaseFootnotesSection = ({
     <>
       <h2>Footnotes</h2>
 
+      <p>
```
Comment (ntsim) [inline src/explore-education-statistics-admin/src/pages/release/edit-release/data/ReleaseFootnotesSection.tsx:85]: We could potentially end up in a state where `p` is empty, i.e. when conditions are true. This isn't great in terms of making the markup semantic.

Wrap each message with `p` to avoid this happening.

File: src/explore-education-statistics-admin/src/pages/release/edit-release/data/ReleaseFootnotesSection.tsx [hunk 3]
```diff
@@ -82,22 +82,34 @@ const ReleaseFootnotesSection = ({
...
+            footnotesData.canUpdateReleaseFootnotes && (
+              <FootnoteForm
+                {...footnoteForm}
+                footnote={undefined}
+                onOpen={footnoteFormControls.create}
+                onCancel={footnoteFormControls.cancel}
+                onSubmit={footnoteFormControls.save}
+                isFirst={
+                  footnotesData.footnotes &&
+                  footnotesData.footnotes.length === 0
```
Comment (ntsim) [inline src/explore-education-statistics-admin/src/pages/release/edit-release/data/ReleaseFootnotesSection.tsx:107]: Could make this statement simpler with `!foonotesData.footnotes?.length`

File: src/explore-education-statistics-admin/src/pages/release/edit-release/data/ReleaseDataPage.tsx (hunk 1 above)
Comment (shakes90) [reply src/explore-education-statistics-admin/src/pages/release/edit-release/data/ReleaseDataPage.tsx:54]: Yeah agreed but there's issues with most of the tabs in the system - often you have to refresh to update status etc. I think all that probably needs addressing on another ticket but for now this is a temp workaround & this will be superseded by the next raft of changes anyway.

File: src/explore-education-statistics-admin/src/pages/release/edit-release/data/ReleaseFootnotesSection.tsx (hunk 3 above)
Comment (shakes90) [reply src/explore-education-statistics-admin/src/pages/release/edit-release/data/ReleaseFootnotesSection.tsx:107]: yep - done - thanks

File: src/explore-education-statistics-admin/src/pages/release/edit-release/data/ReleaseFootnotesSection.tsx (hunk 2 above)
Comment (shakes90) [reply src/explore-education-statistics-admin/src/pages/release/edit-release/data/ReleaseFootnotesSection.tsx:85]: Yep - done - Thanks
